docs/4.3. Controller to Authorization Server Interactions.md

```diff
@@ -242,9 +242,9 @@ Content-Type: application/json
 ```
 
 ## Refreshing Bearer Tokens
-The Bearer Token has a limited life specified by the value of the `expires_in` parameter in seconds. The Node should refresh the token before [it exceeds its half life](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.4._Behaviour_-_Access_Tokens.html#access-token-lifetime) (for example if the token lifetime is 30 seconds, then it should be refreshed at least 15 seconds before token expiry).
+The Bearer Token has a limited life specified by the value of the `expires_in` parameter in seconds. The Controller should refresh the token before [it exceeds its half life](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.4._Behaviour_-_Access_Tokens.html#access-token-lifetime) (for example if the token lifetime is 30 seconds, then it should be refreshed at least 15 seconds before token expiry).
 
-Because refresh tokens have the potential for a long lifetime, developers should ensure that strict storage requirements are in place to keep them from being leaked.
+Because refresh tokens have the potential for a long lifetime, developers should ensure that they are stored securely to keep them from being leaked.
```

Comment:
Perhaps:
>Because refresh tokens have the potential for a long lifetime, it is important that they are stored securely to keep them from being leaked.
to avoid the use of 'should'?

Reply:
```suggestion
Because refresh tokens can have a long lifetime, it is important that they are stored securely to keep them from being leaked.
```